plasmapy/formulary/parameters.py

```diff
@@ -478,10 +483,34 @@ def thermal_speed(
     The particle thermal speed is given by:
 
     .. math::
-        V_{th,i} = \sqrt{\frac{2 k_B T_i}{m_i}}
-
-    This function yields the most probable speed within a distribution
-    function.  However, the definition of thermal velocity varies by
+        V_{th,i} = \sqrt{\frac{N k_B T_i}{m_i}}
+
+    where the value of N depends on the dimensionality and the definition of
+    :math:`v_{th}`: most probable, root-mean-square (RMS), or mean magnitude.
+    The value of N in each case is
+
+    .. list-table:: Values of constant N
+       :widths: 50, 25, 25, 25
+       :header-rows: 1
+
+       * - Dim.
+         - Most-Probable
+         - RMS
+         - Mean-Magnitude
+       * - 1D
+         - 0
+         - 1
+         - :math:`2/\pi`
+       * - 2D
+         - 1
+         - 2
+         - :math:`\pi/2`
+       * - 3D
+         - 2
+         - 3
+         - :math:`8/\pi`
```

Comment:
This is going to be dreadful to read in jupyter/ipython/other shells, but wow, in the rendered docs that is one pretty table. :smile: I'm completely down for that.

Reply:
Yeah - one advantage of calculating these with an explicit dependence on ndim (rather than the if/else tree) would be the ability to more directly state the formulas in the docstring simply for all ndim. 

Side question: how do you get to the docs for a PR (without typing out the link and changing the PR number)?

Reply:
If you look for `docs/readthedocs.org:plasmapy` in the `Checks` tab below, you can click on Details and that should point you to the built docs :)

Reply:
Awesome, thanks!